docs/advanced_topics/customisation/page_editing_interface.rst

```diff
@@ -57,22 +57,99 @@ Wagtail provides a general-purpose WYSIWYG editor for creating rich text content
 
 However, template output from :class:`~wagtail.wagtailcore.fields.RichTextField` is special and need to be filtered to preserve embedded content. See :ref:`rich-text-filter`.
 
-If you're interested in extending the capabilities of the Wagtail WYSIWYG editor (``hallo.js``), See :ref:`extending_wysiwyg`.
+
+.. _rich_text_features:
+
+Limiting features in a rich text field
+--------------------------------------
+
+By default, the rich text editor provides users with a wide variety of options for text formatting and inserting embedded content such as images. However, we may wish to restrict a rich text field to a more limited set of features - for example:
+
+ * The field might be intended for a short text snippet, such as a summary to be pulled out on index pages, where embedded images or videos would be inappropriate;
+ * When page content is defined using :ref:`StreamField <streamfield>`, elements such as headings, images and videos are usually given their own block types, alongside a rich text block type used for ordinary paragraph text; in this case, allowing headings and images to also exist within the rich text content is redundant (and liable to result in inconsistent designs).
+
+This can be achieved by passing a ``features`` keyword argument to ``RichTextField``, with a list of identifiers for the features you wish to allow:
+
+.. code-block:: python
+
+    body = RichTextField(features=['p', 'h2', 'h3', 'b', 'i', 'link'])
+
+The recognised feature identifiers are as follows (note that add-on modules may add to this list):
+
+ * ``p`` - paragraph text (text entered into a rich text field appears in this mode by default, so omitting this from ``features`` is unlikely to be meaningful)
+ * ``h1``, ``h2``, ``h3``, ``h4``, ``h5``, ``h6`` - heading elements
+ * ``b``, ``i`` - bold / italic text
```

Comment:
Doing some research, looking at text editors that already have Wagtail integrations:

- TinyMCE uses `bold` and `italic` (https://www.tinymce.com/docs/configure/content-formatting/#builtinformats) and outputs `strong` and `em` (standalone demo on its site).
- Hallo.js uses `bold` and `italic` (https://github.com/bergie/hallo#plugins), and produces `b` and `i` tags (at least in the standalone demo on its site).
- Draftail (Draft.js) uses `BOLD` / `ITALIC` (https://draftjs.org/docs/advanced-topics-inline-styles.html#mapping-a-style-string-to-css) and produces `strong` and `em`
- Froala uses `bold` and `italic` (https://github.com/jaydensmith/wagtailfroala) and produces `strong` and `em` tags (standalone demo on its site)
- Medium editor uses `bold` and `italic` (https://github.com/dperetti/Django-wagtailmedium) and produces `b` and `i` (standalone demo)

I would suggest switching to `bold` and `italic` for identifiers since it seems to be the most common, it's self-explanatory that it does not match to HTML elements, and the nomenclature for inline styles would scale better if `strikethrough`, `underline`, etc are added as features (either built-in or 3rd-party/custom).

For output – seems like there isn't much consensus. I checked other editors:

- CKEditor produces `strong` and `em` (http://ckeditor.com/)
- Trix produces `strong` and `em` (https://trix-editor.org/)

which seems to indicate `strong` and `em` has the preference. I checked [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong) on the topic, which isn't very helpful in determining the best semantics.